README.md

```diff
@@ -193,6 +191,18 @@ If you specify an array of connections instead of a single connection object Ras
 }
 ```
 
+#### Channel pooling
+Rascal pools channels it uses for publishing messages. It creates a two pools per vhost - one for confirm channels, and other one for "regular" channels. Default maximum pool size for these is 1. If you think it is a limiting factor in your publishing code, the value can be changed through config.
+```json
+{
+  "vhosts": {
+    "v1": {
+      "channelPoolSize": 3
```

Comment:
Maybe we should have separate arguments for confirm and regular. It could get pretty wasteful having a pool of 10 channels which never got used.